docs/mint.json

```diff
@@ -0,0 +1,54 @@
+{
+    "$schema": "https://mintlify.com/schema.json",
+    "name": "Latitude Docs",
+    "logo": {
+      "dark": "/logo/dark.svg",
+      "light": "/logo/light.svg"
+    },
+    "favicon": "/favicon.svg",
+    "colors": {
+      "primary": "#0080ff",
+      "light": "#0080ff",
+      "dark": "#0080ff",
+      "anchors": {
+        "from": "#0080ff",
+        "to": "#0080ff"
+      }
+    },
+    "topbarLinks": [
+      {
+        "name": "",
+        "url": " "
+      }
+    ],
+    "topbarCtaButton": {
+      "name": "Get started",
+      "url": "https://github.com/latitude-dev/latitude-llm"
+    },
+    "primaryTab": {
+      "name": "Guides"
+    },
+    "tabs": [
+      {
+        "name": "Sources",
+        "url": "sources"
+      }
+    ],
+    "anchors": [],
+    "navigation": [
+      {
+        "group": "Getting started",
+        "pages": [
+          "guides/getting-started/introduction",
+          "guides/getting-started/installation",
+          "guides/getting-started/windows-compatibility",
+          "guides/getting-started/troubleshooting"
+        ]
```

Comment:
Windows compatibility xd 👀

Reply:
Yea just temporary in case it needs the structure to not explode